### PR TITLE
Document Node.js Version Support

### PR DESCRIPTION
**Pull request by julianofirme**
### Description
Addresses issue #1 by documenting supported Node.js versions and enforcing version requirements during installation.

### Changes Made
- Added Node.js version documentation to `README.md`.
- Updated `package.json` with `engines` property.
- Included `.npmrc` file for engine rule enforcement.

### Checklist
- [x] Updated `README.md`.
- [x] Modified `package.json`.
- [x] Added `.npmrc` file.
- [x] Reviewed changes.

### Additional Notes
These changes clarify supported Node.js versions and enforce version requirements during installation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ Running `nscli` in your terminal will present you with the following options:
 
 Navigate the list using the arrow keys, then press Enter to run the selected script.
 
+## Node.js Support
+
+This package is compatible with Node versions higher than 16.x
```
Comment (thales-maciel) [inline README.md]: I just ran with node 15.14 without issues

**Review by thales-maciel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (julianofirme) [reply README.md]: I found this doc about the [||= operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment), compatible with Node.js >=15.

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "typescript": "^5.4.5"
   },
   "engines": {
-    "node": ">=16.0.0"
+    "node": "<=15.0.0"
```
Comment (thales-maciel) [inline package.json]: this means we only run on 15 or older, I think you meant `>=` instead of `<=`

**Review by thales-maciel — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (julianofirme) [reply package.json]: oops, fixed.

**Review by thales-maciel — APPROVED** [review]
lgtm

**Pull request merged**